phoenix-core/src/test/java/org/apache/phoenix/parse/QueryParserTest.java

```diff
@@ -289,24 +289,6 @@ public void testNegativeCountStar() throws Exception {
     }
 
     @Test
-    public void testUnknownFunction() throws Exception {
```

Comment:
Is there a reason why this test was removed? Are we no longer throwing an error? Or has the exception code changed? I would think if the user doesn't have load udf enabled then this error is still valid.